VIKINGOS-ANALYZER-V1.0.md

```diff
@@ -0,0 +1,337 @@
+#VIKINGOS ANALYZER V1.0
```

Comment:
markdown formatting does not display correctly, please use this page to validate your preview.

https://www.digitalocean.com/community/markdown?md=

Tip: When you add a # please add a space to see the title correctly
example:
"#VIKINGOS ANALYZER V1.0" ❌
"# VIKINGOS ANALYZER V1.0" ✅